packages/client/src/comp/AddressForm.tsx

```diff
@@ -13,6 +13,7 @@ import { useParams } from 'react-router-dom'
 import { useAppHistory } from '../lib/path'
 import styled from 'styled-components'
 import { sampleAddresses, ImplementedState } from '../common'
+import { StyledModalContext } from './ModalContext'
```

Comment:
I don't think I understand why there's a `useContext(StyledModalContext)` and a `useModal`?  Aren't they doing the same thing and just proving `handleModal` `modal` pairs?

Reply:
well if you look at the react context api with hooks https://reactjs.org/docs/hooks-reference.html#usecontext
this is the correct usage and then you can access functions of that Context